NewForce-Cohort-8/tabloidfullstack-the-armbar-alliance
# Cmv1 comments - List all comments

Trello ticket link: 
https://trello.com/c/nUQQmEoh/19-19-view-a-posts-comments

NEW FEATURE: List Comments

Testing Instructions Steps
Clone this project’s repository to your machine
If you’ve already done this and are currently working on a different branch of the project, be sure to git add and git commit whatever you’re working on on your branch, before you attempt to switch branches
Then git checkout main to switch to the project’s main branch
Navigate to the root directory for this repo (yourParentFolderName/tabloidfullstack-the-armbar-alliance)
git fetch -a to retrieve the current remote-tracked branches for the project, you should see a branch named CMv1Comments as one of the available branches
git checkout CMv1Comments to switch to my branch
If you don't see the most recent version of my files, git pull origin CMv1Comments


Open the project in Visual Studio
If you haven’t created the Tabloidfullstack database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly.
First SQL Script for Creating DB
https://github.com/NewForce-Cohort-8/tabloidfullstack-the-armbar-alliance/blob/main/SQL/01_Tabloid_Create_DB.sql
ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: Second SQL Script for Seeding DB
https://github.com/NewForce-Cohort-8/tabloidfullstack-the-armbar-alliance/blob/main/SQL/02_Tabloid_Seed_Data.sql
Click the execute with debugger solid green button to launch the app
In your browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com) - this should redirect you back to the home page.
At the homepage click on the Posts link in the nav bar.
You should see a Posts Listed on this screen.
You will then click the title of a post revealing the details and comments of the post.

The comment list should display in dated order newest first. 

Details will show Username, Date posted, Comment subject, and Comment content

Checklist:
[]My code follows the style guidelines of this project
[]I have performed a self-review of my own code
[]I have commented my code, particularly in hard-to-understand areas
[]My changes generate no new warnings or errors
[]I have added test instructions that prove my fix is effective or that my feature works
